frontend/test/src/container/Account.spec.js

```diff
@@ -31,7 +31,7 @@ import {
   mockPutMyselfName200,
   mockPutMyselfPassword204, mockPutMyselfPassword403
 } from '../../apiMock'
-
+// TODO update tests
```

Comment:
if you plan on updating the tests in that PR, no problem.
If you plan on doing it later with another issue, don't forget the link to the issue

Reply:
in this PR, just a self reminder :smile: